crates/aptos/src/account/multisig_account.rs

```diff
@@ -0,0 +1,244 @@
+    #[clap(long)]
+    pub(crate) num_signatures_required: u64,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+}
+
+/// A shortened create multisig account output
+#[derive(Clone, Debug, Serialize)]
+pub struct CreateSummary {
+    pub multisig_account: Option<AccountAddress>,
+    #[serde(flatten)]
+    pub transaction_summary: TransactionSummary,
+}
+
+impl From<Transaction> for CreateSummary {
+    fn from(transaction: Transaction) -> Self {
+        let transaction_summary = TransactionSummary::from(&transaction);
+
+        let mut summary = CreateSummary {
+            transaction_summary,
+            multisig_account: None,
+        };
+
+        if let Transaction::UserTransaction(txn) = transaction {
+            summary.multisig_account = txn.info.changes.iter().find_map(|change| match change {
+                WriteSetChange::WriteResource(WriteResource { address, data, .. }) => {
+                    if data.typ.name.as_str() == "Account"
+                        && *address.inner().to_hex() != *txn.request.sender.inner().to_hex()
+                    {
+                        Some(*address.inner())
+                    } else {
+                        None
+                    }
+                },
+                _ => None,
+            });
+        }
+
+        summary
+    }
+}
+
+#[async_trait]
+impl CliCommand<CreateSummary> for Create {
+    fn command_name(&self) -> &'static str {
+        "Create"
+    }
+
+    async fn execute(self) -> CliTypedResult<CreateSummary> {
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_create_with_owners(
+                self.additional_owners,
+                self.num_signatures_required,
+                vec![],
+                vec![],
+            ))
+            .await
+            .map(CreateSummary::from)
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct CreateTransaction {
+    #[clap(long, parse(try_from_str=crate::common::types::load_account_arg))]
+    pub(crate) multisig_account: AccountAddress,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+
+    /// Function name as `<ADDRESS>::<MODULE_ID>::<FUNCTION_NAME>`
+    ///
+    /// Example: `0x842ed41fad9640a2ad08fdd7d3e4f7f505319aac7d67e1c0dd6a7cce8732c7e3::message::set_message`
+    #[clap(long)]
+    pub(crate) function_id: MemberId,
+
+    /// Arguments combined with their type separated by spaces.
+    ///
+    /// Supported types [u8, u16, u32, u64, u128, u256, bool, hex, string, address, raw, vector<inner_type>]
+    ///
+    /// Example: `address:0x1 bool:true u8:0 u256:1234 'vector<u32>:a,b,c,d'`
+    #[clap(long, multiple_values = true)]
+    pub(crate) args: Vec<ArgWithType>,
+
+    /// TypeTag arguments separated by spaces.
+    ///
+    /// Example: `u8 u16 u32 u64 u128 u256 bool address vector signer`
+    #[clap(long, multiple_values = true)]
+    pub(crate) type_args: Vec<MoveType>,
```

Comment:
We should make these into a common type so we share it with the run function / not duplicate logic.